cmd/example-mysql/main.go

```diff
@@ -132,33 +132,35 @@ func main() {
 // "/tile/0/x001/x234/067.p/8" means level 0, index 1234067 and width 8 of a partial tile.
 func parseTileLevelIndexWidth(level, index string) (uint64, uint64, uint64, error) {
 	l, err := strconv.ParseUint(level, 10, 64)
-	if err != nil {
+	if l > 63 || err != nil {
 		return 0, 0, 0, fmt.Errorf("failed to parse tile level")
 	}
 
-	var i, w uint64
-	switch indexPaths := strings.Split(index, "/"); len(indexPaths) {
-	// Full tile = 3
-	// Partial tile = 4
-	case 3, 4:
-		indexPath := strings.Join(indexPaths[0:3], "")
-		indexPath = strings.ReplaceAll(indexPath, "x", "")
-		indexPath = strings.ReplaceAll(indexPath, ".p", "")
-		i, err = strconv.ParseUint(indexPath, 10, 64)
-		if err != nil {
+	i := uint64(0)
```

Comment:
nit: move this declaration down to above for `for` loop on 156 to increase locality? I was reading the loop down there and then had to search for where `i` was defined.

Reply:
Done.